src/app/services/user/user.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { User } from '../../shared/models/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  private apiUrl = 'https://rb-messenger.azurewebsites.net/users/';
+
+  constructor(private http: HttpClient) { }
+
+  getUsers(numberOfUsers: number, page: number, valueContains: string,
```

Comment:
If you have more than 3 params passed to the method/func, you should create an Object